src/controllers/itemdetailpage.js

```diff
@@ -62,7 +62,7 @@ define(['loading', 'appRouter', 'layoutManager', 'connectionManager', 'userSetti
 
     function getProgramScheduleHtml(items) {
         var html = '';
-        html += '<div is="emby-itemscontainer" class="itemsContainer vertical-list" data-contextmenu="false">';
+        html += '<div is='emby-itemscontainer' class='itemsContainer vertical-list' data-contextmenu='false'>';
```

Comment:
sorry, but this change is even worse than before %) it won't work!

Reply:
I probably should stop using Notepad++ to make my changes. It seems to mess up a lot of things.

Reply:
I started with a fresh copy of the file right out of the current master branch and lots of errors are coming up - all things I haven't touched. What's going on here?

Reply:
You likely need our ESLint config.

Reply:
Clearing the IDE cache solved the "problems". Strange things happen in software world...